### PR TITLE
SONAR-7027 add DevCockpitBridge and start/stop Dev Cockpit plugin

### PR DESCRIPTION
**Pull request by sns-seb**
add Maven module sonar-dev-cockpit-bridge

### CODE REVIEW

File: server/sonar-dev-cockpit-bridge/src/main/java/org/sonar/server/devcockpit/DevCockpitBridge.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import org.sonar.core.platform.ComponentContainer;
+
+/**
+ * Interface implemented by the Extension point exposed by the Developer Cockpit plugin that serves as the unique access
+ * point from the whole SQ instance into the Developer Cockpit plugin.
+ */
+public interface DevCockpitBridge {
+
+  /**
+   * Bootstraps the Views plugin.
```
Comment (julienlancelot) [inline server/sonar-dev-cockpit-bridge/src/main/java/org/sonar/server/devcockpit/DevCockpitBridge.java]: Bad copy paste : Views -> Deveveloper Cockpit

Comment (sns-seb) [inline server/sonar-dev-cockpit-bridge/src/main/java/org/sonar/server/devcockpit/DevCockpitBridge.java]: yes

**Comment by julienlancelot** [comment]
Be careful, build is failing because of missing header

File: server/sonar-dev-cockpit-bridge/src/main/java/org/sonar/server/devcockpit/package-info.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+@ParametersAreNonnullByDefault
+package org.sonar.server.views;
```
Comment (SonarTech) [inline server/sonar-dev-cockpit-bridge/src/main/java/org/sonar/server/devcockpit/package-info.java]: MAJOR This file "package-info.java" should be located in "org/sonar/server/views" directory, not in "/home/travis/build/SonarSource/sonarqube/server/sonar-dev-cockpit-bridge/src/main/java/org/sonar/server/devcockpit". [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1598)

**Comment by SonarTech** [comment]
SonarQube analysis reported 4 issues:
* MAJOR 4 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'protocol' package (org.sonarsource.sonarqube:sonar-batch-protocol:target/generated-sources/protobuf/org/sonar/batch/protocol) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'input' package (org.sonarsource.sonarqube:sonar-batch-protocol:target/generated-sources/protobuf/org/sonar/batch/protocol/input) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'output' package (org.sonarsource.sonarqube:sonar-batch-protocol:target/generated-sources/protobuf/org/sonar/batch/protocol/output) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)

**Comment by sns-seb** [comment]
@julienlancelot build is now ok and DevCockpit fix for batch execution is available in PR on DevCockpit side

File: server/sonar-dev-cockpit-bridge/src/main/java/org/sonar/server/devcockpit/package-info.java (hunk 2 above)
Comment (julienlancelot) [inline server/sonar-dev-cockpit-bridge/src/main/java/org/sonar/server/devcockpit/package-info.java]: Do not forget to replace the package name

**Comment by julienlancelot** [comment]
Indeed, it's working, LGTM !

**Pull request merged**